cmake/std/PullRequestLinuxCuda9.2TestingSettings.cmake

```diff
@@ -121,4 +123,6 @@ set (KokkosCore_UnitTest_Cuda_MPI_1_EXTRA_ARGS
   "--gtest_filter=-cuda.debug_pin_um_to_host:cuda.debug_serial_execution"
   CACHE STRING "Temporary disable for CUDA PR testing")
 
+set(CMAKE_CXX_FLAGS "-std=c++14" CACHE STRING "Set standard to C++14")
```

Comment:
We shouldn't be explicitly adding this to the CXXFLAGS. @bartlettroscoe I believe we fixed this so that we just set the standard?

Reply:
Yes, TriBITS, and therefore Trilinos, should just be using the CMake C++ standards support as per https://github.com/trilinos/Trilinos/pull/6814, https://github.com/TriBITSPub/TriBITS/pull/300.

Reply:
I'll remove it from this build.
It has made it into all of the other builds. I'll go back and write another PR to remove it from all of the rest.